### ISSUE TITLE
Test app - Links to manager instance should honor MANAGER_HOSTNAME variable

### ISSUE THREAD

**Issue by pini-gh** [LemonLDAPNG/lemonldap-ng-docker#35]
Hi,

While playing with this docker instance I noticed that the test app seems to have the 'manager' hostname hardcoded. Then the presented links to the manager instance are broken if ${MANAGER_HOSTNAME} is different from 'manager.${SSO_DOMAIN}'.

Thanks.

**Comment by coudot** [comment]
Indeed, we should edit the test index.pl to replace hostnames when running the image.
